src/lib/services/groups/service/groups.ts

```diff
@@ -13,21 +12,21 @@ import type {
  * @see https://groups.api.globus.org/redoc#tag/groups/operation/get_my_groups_and_memberships_v2_groups_my_groups_get
  */
 export const getMyGroups = function (
-  options = {},
+  options?,
```

Comment:
Does this work with the first argument being optional? If you only pass one argument, how do we know it it's meant to be `options` or `sdkOptions`? This feels like something I should know - maybe it's early :sleepy:

Reply:
Both arguments are optional; You can call this method as `groups.groups.getMyGroups()`). The first argument will always be treated as `options`. If you wanted to call this with only `sdkOptions` you still have to appease the function arity;`groups.groups.getMyGroups(undefined, { ... })`